### PR TITLE
Work page

### PR DESCRIPTION
**Pull request by cmwall**
This commit creates the work page.

### CODE REVIEW

**Comment by cmwall** [comment]
This is ready for review. We need to add the project images but I think @Connorelsea already has those component built. They just aren't merged yet.

File: src/elements/textStyles.js [hunk 1]
```diff
@@ -3,29 +3,48 @@ import glamorous from "glamorous"
...
-  <HeroStyle {...props}>{children}</HeroStyle>
+function removeUndefined(obj) {
+  for (let k in obj) if (obj[k] === undefined) delete obj[k]
+  return obj
+}
+
+function appliedStyles(styleObject, applied) {
+  return {
+    ...styleObject,
+    ...removeUndefined(applied),
```
Comment (Croge32) [inline src/elements/textStyles.js]: Why is this necessary?

File: src/pages/work.js [hunk 2]
```diff
@@ -1,3 +1,139 @@
...
+              <HashListItem>Prototyping</HashListItem>
+            </HashListColumn>
+            <HashListColumn>
+              <HashListItem>Web Development</HashListItem>
+              <HashListItem>React & React Native</HashListItem>
+              <HashListItem>JavaScript</HashListItem>
+            </HashListColumn>
+            <HashListColumn>
+              <HashListItem>Mobile Development</HashListItem>
+              <HashListItem>Frameworks???</HashListItem>
```
Comment (Croge32) [inline src/pages/work.js]: I think you can remove these question marks.

File: src/pages/work.js [hunk 3]
```diff
@@ -1,3 +1,139 @@
...
+    <CenterContainer>
+      <HeaderOne>Some projects we've worked on.</HeaderOne>
+      <Body>
+        We work with clients large and small. Our work is backed by research and
+        utilizes the latest technology. Our process is custom-tailored for each
+        project to ensure we're creating the best possible product.
+      </Body>
+
+      <HashList>
+        <Media query="(min-width: 625px)">
```
Comment (Croge32) [inline src/pages/work.js]: Are these max width/min width values going to be reused in other places on the app?

**Comment by Croge32** [comment]
You need to reduce font-size for the Hero stuff on smaller screen sizes:

screen shot 2017-11-21 at 10 53 37 am

File: src/elements/textStyles.js (hunk 1 above)
Comment (cmwall) [reply src/elements/textStyles.js]: So some JS trivia:

```js
Object.assign(
  { },
  { color: "red" },
  { color: undefined }
)
```

That will set color to undefined! Not what we want here. We want to overwrite color if it is passed in. Otherwise, we want a default color. This method removes that undefined value from the object before we set it.

File: src/pages/work.js (hunk 2 above)
Comment (cmwall) [reply src/pages/work.js]: Ok so I added that there because I didn't know if we really needed that section here. It's weird text compared to everything else. 

We specify "Ruby on Rails development", "React development", etc. and then we just have "Frameworks" which does not make sense.

File: src/pages/work.js (hunk 3 above)
Comment (cmwall) [reply src/pages/work.js]: No, those are specifically for this list component to lay out at different breakpoints.

File: src/pages/work.js (hunk 2 above)
Comment (cmwall) [reply src/pages/work.js]: We decided to go with "Consultation" instead.
